src/Tempest/Console/src/Commands/Generators/MakeControllerCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Commands\Generators;
+
+use Tempest\Console\ConsoleArgument;
+use Tempest\Console\ConsoleCommand;
+use Tempest\Console\Stubs\ControllerStub;
+use Tempest\Generation\HasGeneratorCommand;
+
+final class MakeControllerCommand
+{
+    use HasGeneratorCommand;
```

Comment:
Let's rename to `IsGeneratorCommand`

Reply:
I've named it like so because it can be a class that holds multiple console commands on multiple methods.
I think this is also why we currently had the `HasConsole` trait and not a `IsConsoleCommand`.
What do you think ?